socorro/mozilla_rulesets.py

```diff
@@ -0,0 +1,121 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+from socorro import settings
```

Comment:
This was all in `socorro/processor/processor_pipeline.py` and I moved it to here.